src/bonsai/bonsai/tool/cost.py

```diff
@@ -584,45 +584,71 @@ def import_cost_schedule_csv(
         import time
 
         start = time.time()
-        csv2ifc = Csv2Ifc(file_path, tool.Ifc.get(), is_schedule_of_rates=is_schedule_of_rates)
+
+        resolved_path = tool.Ifc.resolve_uri(file_path)
+
+        csv2ifc = Csv2Ifc(resolved_path, tool.Ifc.get(), is_schedule_of_rates=is_schedule_of_rates)
         csv2ifc.execute()
         print("Import finished in {:.2f} seconds".format(time.time() - start))
         return csv2ifc.cost_schedule
 
     @classmethod
-    def add_csv_filepath(
-        cls,
-        file_path: Optional[str] = None,
-        is_schedule_of_rates: bool = False,
-        cost_schedule: ifcopenshell.entity_instance = None,
-    ) -> None:
-        if not file_path:
+    def add_csv_filepath(cls, file_path: str, is_schedule_of_rates: bool, cost_schedule) -> None:
```

Comment:
@falken10vdl why have you deleted the type checking? I don't know so much about it but i think it could be useful...